dashboard-widgets.el

```diff
@@ -370,10 +375,13 @@ nil then no banner is displayed."
 Will be of the form `(list-type . list-function)'.
 Possible values for list-type are: `recents', `bookmarks', `projects',
 `agenda' ,`registers'."
-  :type  '(repeat (alist :key-type symbol :value-type function))
+  :type  '(alist :key-type symbol :value-type function)
   :group 'dashboard)
 
-(defcustom dashboard-projects-backend 'projectile
+(defcustom dashboard-projects-backend
+  (if (fboundp 'projectile-find-file)
```

Comment:
I don't think it is a good idea to "guess" custom variables, maybe it's time to change the default value to `project-el` because now it's included in Emacs. This would be another PR.

Reply:
I agree with @ricardoricho. I'm using the built-in `project.el` as well.